corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserMain.java

```diff
@@ -0,0 +1,81 @@
+package org.corfudb.browser;
+
+import java.util.Map;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.CorfuDynamicKey;
+import org.corfudb.runtime.collections.CorfuDynamicRecord;
+import org.corfudb.runtime.collections.CorfuTable;
+import org.corfudb.util.GitRepositoryState;
+import org.docopt.Docopt;
+
+/**
+ * Main class for the CorfuTable Browser Tool.
+ * Command line options are documented in the USAGE variable.
+ *
+ * - Created by pmajmudar on 10/16/2019
+ */
+@Slf4j
+public class CorfuStoreBrowserMain {
+    private static final String USAGE = "Usage: corfu-browser --host=<host> " +
+        "--port=<port> --namespace=<namespace> --tablename=<tablename> " +
+        "[--keystore=<keystore_file>] [--ks_password=<keystore_password>] " +
+        "[--truststore=<truststore_file>] [--truststore_password=<truststore_password>] " +
+        "[--tlsEnabled=<tls_enabled>]\n"
+        + "Options:\n"
+        + "--host=<host>   Hostname\n"
+        + "--port=<port>   Port\n"
+        + "--namespace=<namespace>   Namespace\n"
+        + "--tablename=<tablename>   Table Name\n"
+        + "--keystore=<keystore_file> KeyStore File\n"
+        + "--ks_password=<keystore_password> KeyStore Password\n"
+        + "--truststore=<truststore_file> TrustStore File\n"
+        + "--truststore_password=<truststore_password> Truststore Password\n"
+        + "--tlsEnabled=<tls_enabled>";
+
+    public static void main(String[] args) {
+        CorfuRuntime runtime;
+        // Parse the options given, using docopt.
+        Map<String, Object> opts =
+            new Docopt(USAGE)
+                .withVersion(GitRepositoryState.getRepositoryState().describe)
+                .parse(args);
+        String host = opts.get("--host").toString();
+        Integer port = Integer.parseInt(opts.get("--port").toString());
+        boolean tlsEnabled = Boolean.parseBoolean(opts.get("--tlsEnabled").toString());
+
+        if (tlsEnabled) {
+            String keystore = opts.get("--keystore").toString();
+            String ks_password = opts.get("--ks_password").toString();
+            String truststore = opts.get("--truststore").toString();
+            String truststore_password =
+                opts.get("--truststore_password").toString();
+            runtime = CorfuRuntime.fromParameters(
+                CorfuRuntime.CorfuRuntimeParameters.builder().useFastLoader(false)
```

Comment:
Start the Parameters builder outside the if loop and build() it after the else to initialize the runtime.
You can then set duplicate options like fast loader and cache disabled only once above.